packages/lesswrong/integrationTests/sessions.tests.ts

```diff
@@ -125,10 +125,10 @@ describe("Sessions", () => {
       state.store.get(sid, (error, session) => {
         expect(error).toBeNull();
         expect(session).toEqual(serializedSession);
-        done();
+        done()
       });
     });
-    state.store.set(sessionId, sessionData);
+    void state.store.set(sessionId, sessionData);
```

Comment:
Wait... does this mean the linter doesn't run on integration tests? That's a bit iffy.

Reply:
I don't thinks so. Previously this was not actually an async function, it just dispatched an async function internally, so there was no need for a `void`